src/button.tsx

```diff
@@ -13,7 +13,14 @@ import {eventActions, eventCategories, eventNames, useTrackingConfig} from './ut
 import {useTheme} from './hooks';
```

Comment:
better review this ignoring whitespace: https://github.com/Telefonica/mistica-web/pull/551/files?w=1